experimental/dds/tree2/src/test/feature-libraries/editable-tree/mockData.ts

```diff
@@ -41,96 +40,66 @@ import { brand, Brand } from "../../../util";
+	),
 );
 
-export const globalFieldSymbolSequencePhones: GlobalFieldKeySymbol =
-	globalFieldSchemaSequencePhones.symbol;
-
-export const addressSchema = builder.object("Test:Address-1.0.0", {
-	local: {
-		zip: SchemaBuilder.field(FieldKinds.value, stringSchema, int32Schema),
-		street: SchemaBuilder.field(FieldKinds.optional, stringSchema),
-		city: SchemaBuilder.field(FieldKinds.optional, stringSchema),
-		country: SchemaBuilder.field(FieldKinds.optional, stringSchema),
-		phones: SchemaBuilder.field(FieldKinds.optional, phonesSchema),
-		sequencePhones: SchemaBuilder.field(FieldKinds.sequence, stringSchema),
-	},
-	globalFields: [globalFieldSchemaSequencePhones],
+export const addressSchema = builder.struct("Test:Address-1.0.0", {
+	zip: SchemaBuilder.field(FieldKinds.value, stringSchema, int32Schema),
+	street: SchemaBuilder.field(FieldKinds.optional, stringSchema),
+	city: SchemaBuilder.field(FieldKinds.optional, stringSchema),
+	country: SchemaBuilder.field(FieldKinds.optional, stringSchema),
+	phones: SchemaBuilder.field(FieldKinds.optional, phonesSchema),
+	sequencePhones: SchemaBuilder.field(FieldKinds.sequence, stringSchema),
 });
 
-export const mapStringSchema = builder.object("Map<String>", {
-	extraLocalFields: SchemaBuilder.field(FieldKinds.optional, stringSchema),
-	value: ValueSchema.Serializable,
-});
+export const mapStringSchema = builder.map(
+	"Map<String>",
+	SchemaBuilder.field(FieldKinds.optional, stringSchema),
+);
 
-export const personSchema = builder.object("Test:Person-1.0.0", {
-	local: {
-		name: SchemaBuilder.field(FieldKinds.value, stringSchema),
-		age: SchemaBuilder.field(FieldKinds.optional, int32Schema),
-		adult: SchemaBuilder.field(FieldKinds.optional, boolSchema),
-		salary: SchemaBuilder.field(FieldKinds.optional, float64Schema, int32Schema, stringSchema),
-		friends: SchemaBuilder.field(FieldKinds.optional, mapStringSchema),
-		address: SchemaBuilder.field(FieldKinds.optional, addressSchema),
-	},
+export const personSchema = builder.struct("Test:Person-1.0.0", {
+	name: SchemaBuilder.field(FieldKinds.value, stringSchema),
+	age: SchemaBuilder.field(FieldKinds.optional, int32Schema),
+	adult: SchemaBuilder.field(FieldKinds.optional, boolSchema),
+	salary: SchemaBuilder.field(FieldKinds.optional, float64Schema, int32Schema, stringSchema),
+	friends: SchemaBuilder.field(FieldKinds.optional, mapStringSchema),
+	address: SchemaBuilder.field(FieldKinds.optional, addressSchema),
 });
 
-export const optionalChildSchema = builder.object("Test:OptionalChild-1.0.0", {
-	local: {
-		child: SchemaBuilder.fieldOptional(Any),
-	},
-	value: ValueSchema.Serializable,
+export const optionalChildSchema = builder.struct("Test:OptionalChild-1.0.0", {
+	child: SchemaBuilder.fieldOptional(Any),
 });
 
-export const arraySchema = builder.object("Test:Array-1.0.0", {
-	local: {
-		[EmptyKey]: SchemaBuilder.field(FieldKinds.sequence, stringSchema, int32Schema),
-	},
-});
+export const arraySchema = builder.fieldNode(
```

Comment:
`fieldNode` feels a little strange to me. The usage for arrays makes sense but are there other scenarios where we'd prefer this over struct or terminal?

Reply:
If not, I think we should just support array directly.

Reply:
I have added some more docs.